tools/serverlocalizer/package.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "serverlocalizer",
+  "version": "1.0.0",
+  "description": "The localizator of countries and cities",
```

```suggestion
  "description": "Tool to localize country and city names",
```

"localizator" really sounds like a stange movie character ;)